src/main/resources/com/google/api/codegen/copyright-google.txt

```diff
@@ -1 +1 @@
-Copyright 2018 Google LLC
+Copyright 2018 Google LLC
```

Comment:
I'm confused what was changed here?

Reply:
There used to be a newline at the end of the file, now it's removed.